pom.xml

```diff
@@ -126,13 +126,14 @@
                     <version>${org.codehaus.mojo.sql-maven-plugin.version}</version>
                 </plugin>
                 <plugin>
-                    <groupId>net.revelc.code</groupId>
+                    <groupId>net.revelc.code.formatter</groupId>
                     <artifactId>formatter-maven-plugin</artifactId>
                     <version>${formatter-maven-plugin.version}</version>
                     <configuration>
                         <configFile>${project.root.basedir}/eclipse/formatter.xml
                         </configFile>
                         <lineEnding>CRLF</lineEnding>
+                        <sourceDirectory>${project.basedir}</sourceDirectory>
```

Comment:
- [ ] Is this reasonable? This is not specified the correct source directory

For example, when you execute `mvn formatter:format -pl terasoluna-gfw-functionaltest-domain`
* correct source directory: `[root]/terasoluna-gfw-functionaltest-domain/src/main/java`
* actually specified: `[root]/terasoluna-gfw-functionaltest-domain`